forms/user.form.php

```diff
@@ -99,8 +99,15 @@
 		$error['password'] = 'Enter your Password';
 	}
 
+	if (empty($password) || strlen($password) < 8) {
+		throw new InvalidArgumentException("Password cannot be empty and must be at least 8 characters");
+	}
+
+	// here we want to hash the pw and save the hash in the db
+	$hash = password_hash($password, PASSWORD_BCRYPT);
+
 	// creating a new user object
-	$user = new User($firstName, $lastName, $email, $password, $tel, false);
+	$user = new User($firstName, $lastName, $email, $hash, $tel, false);
```

Comment:
Entered a password that was longer than 8 characters and received the following error message.
![image](https://user-images.githubusercontent.com/78501210/112517844-2feed680-8d6f-11eb-8bf8-d54a00e0bd8f.png)

Entered an 8 character password and received this error message
![image](https://user-images.githubusercontent.com/78501210/112518047-67f61980-8d6f-11eb-8d0c-b3b09ceda093.png)

Reply:
@vricci518 you didnt run the sql update. See the pr description.